### DISCUSSION TITLE
fa/manager/domain-worker-cdn-and-tunneling/How-to-use-Amazon-CDN/

### DISCUSSION THREAD

**Discussion by giscus** [hiddify/Hiddify-Manager#4232] [Content Help کمک به اصلاح محتوا]
# fa/manager/domain-worker-cdn-and-tunneling/How-to-use-Amazon-CDN/

The proxy manager platform.

https://hiddify.com/fa/manager/domain-worker-cdn-and-tunneling/How-to-use-Amazon-CDN/

**Comment by roodan52** [comment]
سلام
کلا این خطا رو می گیرم
هیچ جوره cname رو قبول نمیکنه

To add an alternate domain name (CNAME) to a CloudFront distribution, you must attach a trusted certificate that validates your authorization to use the domain name. For more details, see: https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/CNAMEs.html#alternate-domain-names-requirements

**Reply by TomCat-Hub** [reply]
درود بر شما
وقت بخیر

اگر میخواید از آمازون کلادفرانت برای کانفیگ CDN استفاده کنید خیلی راحت میتونید مستقیما دامنه ی خود کلادفرانت رو به عنوان دامنه ی CDN توی هیدیفای منیجر استفاده کنید. اینجور علاوه بر اینکه نیاز نخواهید داشت بصورت دستی سرتیفیکیت برای دامنه بگیرید، دامنه ی خودتون هم از فیلترینگ مصون باقی میمونه. 
اما اگر به هر دلیلی میخواید از دامنه ی خودتون به عنوان دامنه ی CDN استفاده کنید، حتما باید اول برای اون دامنه سرتیفیکیت بگیرید. روند سرت گرفتن توی مقاله بخوبی توضیح داده شده و اگر مراحل رو بدرستی طی کنید خیلی راحت و سریع میتونید برای دامنتون سرتیفیکیت بگیرید و نکته ی خاص و طلایی وجود نداره و خیلی راحته.

لطفا اگر سوال بیشتری داشتید یا نیاز به راهنمایی بیشتری بود توی گروه تلگرام ما پیام بفرستید و مشکلتون رو با ذکر جزییات بیان کنید تا دوستان شما رو راهنمایی کنند.

لینک گروه تلگرام هیدیفای:
https://t.me/hiddify_board

باتشکر از شما
مانا باشید!
